contributingGuides/STYLE.md

```diff
@@ -70,7 +630,7 @@ export {
 
 Using named functions is the preferred way to write a callback method.
```

Comment:
I don't think this rule is followed in the codebase, I think we could remove it.